packages/storybook/src/nimble/tree-view/tree-view.stories.ts

```diff
@@ -33,12 +35,14 @@ interface AnchorItemArgs {
 }
 
 const selectionModeDescription = `
-<li>All: all items in the tree are selectable through user interaction</li>
-<li>Leaves only: only the leaf items in the tree are selectable through user interaction</li>
-<li>None: no items in the tree are selectable through user interaction</li>
+<li>all: all items in the tree are selectable through user interaction</li>
+<li>leaves-only: only the leaf items in the tree are selectable through user interaction</li>
+<li>none: no items in the tree are selectable through user interaction</li>
 <br>
 Note: Changing the selection mode does not affect which items can be selected programmatically.
 `;
+const iconDescription = 'To place an icon at the far-left of the item, set `slot="start"` on the icon.';
```

Comment:
```suggestion
const iconDescription = 'An icon to display to the left of the item text.';
```

Reply:
Switched to use the shared description string instead.